WooCommerce/Resources/en.lproj/Localizable.strings

```diff
@@ -1733,12 +1736,21 @@
    Site Address placeholder */
 "example.com" = "example.com";
 
+/* Button to submit selection on the Exclude Categories screen when more than 1 item is selected. Reads like: Exclude 10 Categories */
+"Exclude %1$d Categories" = "Exclude %1$d Categories";
+
+/* Button to submit selection on the Exclude Categories screen when 1 item is selected */
+"Exclude %1$d Category" = "Exclude %1$d Category";
+
 /* Title of the action button at the bottom of the Exclude Products screen when more than 1 item is selected, reads like: Exclude 5 Products */
 "Exclude %1$d Products" = "Exclude %1$d Products";
 
 /* Title of the action button at the bottom of the Exclude Products screen when one product is selected */
 "Exclude 1 Product" = "Exclude 1 Product";
 
+/* Title for the Exclude Categories screen */
+"Exclude categories" = "Exclude categories";
```

Comment:
Found in the version with the new tooling, too. See [`5c21528`](https://github.com/woocommerce/woocommerce-ios/pull/6737/commits/5c215284d6839f397732f684ec664405f42939ab#diff-96d03942a96e588fcdca0288a2a18d0466a6e44ee5b256f581cd17f2086ea22dR1836-R1837)